fuse_constraints/test/cost_function_gtest.h

```diff
@@ -95,20 +95,20 @@ static void ExpectCostFunctionsAreEqual(const ceres::CostFunction& cost_function
   EXPECT_TRUE(actual_cost_function.Evaluate(parameter_blocks.get(), actual_residuals.get(), nullptr));
   for (size_t i = 0; i < num_residuals; ++i)
   {
-    EXPECT_NEAR(residuals[i], actual_residuals[i], tolerance) << "residual id: " << i;
+    EXPECT_DOUBLE_EQ(residuals[i], actual_residuals[i]) << "residual id: " << i;
```

Comment:
Is `EXPECT_DOUBLE_EQ` more tolerant than 1e-18?

Reply:
yep - it knows what the underlying buckets of precision are for doubles